### PR TITLE
done

### PR DESCRIPTION
**Pull request by VladyslavKucher**
- [DEMO LINK](https://VladyslavKucher.github.io/layout_html-form/)
- [TEST REPORT LINK](https://VladyslavKucher.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

**Review by qoa11a — CHANGES_REQUESTED** [review]
please, provide working test report link

**Comment by VladyslavKucher** [comment]
https://VladyslavKucher.github.io/layout_html-form/report/html_report/
This is a corect test report link. However, it is not working. It would be highly appreciated, if you could tell me how can I fix it:)
If this test created to check html linter, below you will se attached screenshot of "npm run lint"
image

**Comment by VladyslavKucher** [comment]
file:///C:/Users/MyHome/Desktop/projects/projectLayoutHtmlForm/layout_html-form/backstop_data/html_report/index.html

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,166 @@
...
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-field block">
+        <legend>Personal information</legend>
+        <div class="input">
+          Surname:
+          <input type="text" 
+          name="attribute" 
+          autocomplete="off" 
+          required
+          >
```
Comment (polosanya) [inline src/index.html:24]:
```suggestion
          <input 
            type="text" 
            name="attribute" 
            autocomplete="off" 
            required
          >
```
Fix all cases
image

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,166 @@
...
+        </div>
+
+        <div class="input">
+          Name:
+          <input type="text" 
+          name="attribute" 
+          autocomplete="off" 
+          required
+          >
+        </div>
```
Comment (polosanya) [inline src/index.html:34]: image

**Review by polosanya — CHANGES_REQUESTED** [review]
image
Test link is still not working.
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster because your answers in GitHub will be visible to the mentor only after you re-request review 🙂

**Pull request closed without merging**